LoRaEngine/LoraKeysManagerFacade/IoTCentralImp/DeviceTwin.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade.IoTCentralImp
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    public class DeviceTwin : IDeviceTwin
+    {
+        private readonly DesiredProperties properties;
+
+        public DeviceTwin(string deviceName, DesiredProperties properties)
+        {
+            this.DeviceId = deviceName;
+            this.properties = properties;
+        }
+
+        public string DeviceId { get; }
+
+        public string GetDevAddr()
```

Comment:
you can simplify a bit all those single line returns:

```csharp
public string GetDevAddr() => this.properties.DevAddr;
```

I makes it more concise and easy then to read.